custom-login/src/app/app.config.ts

```diff
@@ -8,7 +9,8 @@ export default {
     scopes: ['openid', 'profile', 'email'],
     testing: {
       disableHttpsCheck: `${OKTA_TESTING_DISABLEHTTPSCHECK}`
-    }
+    },
+    useInteractionCodeFlow: `${USE_INTERACTION_CODE}`,
```

Comment:
For a boolean value, no need for string template literal

```
 useInteractionCodeFlow: USE_INTERACTION_CODE
```

(if you know it is boolean)


```
 useInteractionCodeFlow: !!USE_INTERACTION_CODE
```

(will convert to boolean)